server/athenian/api/openapi/openapi.yaml

```diff
@@ -1056,3 +1085,163 @@ components:
       - date_from
       - date_to
       type: object
+    FilterPullRequestsRequest:
+      example:
+        account: 1
+        date_from: 2000-01-23
+        date_to: 2020-05-23
+        in:
+        - '{1}'
+        stages:
+        - wip
+      properties:
+        account:
+          description: Session account ID.
+          type: integer
+        date_from:
+          description: PRs must be updated later than or equal to this date.
+          format: date
+          type: string
+        date_to:
+          description: PRs must be updated earlier than or equal to this date.
```

Comment:
The "updated" date consider either adding commits or adding comments?

Reply:
Any action that influences the stage assignment.

Reply:
So... if I understood you, a PR that was approved (in "merge" stage) more than 1 month ago, but is stale, and there are comments asking for merge, won't appear in queries filtering by the last month (because that comments won't change the stage, which is already in "approve".
Is that correct?

Even more:
Filtering by the last 30 days, there won't appear open PRs that didn't transition between stages in the last 30 daysbe. Isn't it unexpected? wdyt @warenlg ?

Reply:
Nope, it should appear in the queries. the current logic is complex as hell, but it should take care of such cases. IDK how to express that huge pile of complex shit that I mentioned on Slack in a few sentences.

However, they will not participate in metrics calculation because we don know the exact **ending datetime** when the stage changed.